src/custom.rs

```diff
@@ -103,3 +103,39 @@ pub fn getrandom_inner(dest: &mut [MaybeUninit<u8>]) -> Result<(), Error> {
         Some(code) => Err(Error::from(code)),
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    const LEN7_CODE: u32 = Error::CUSTOM_START + 7;
+    // Returns a custom error if input is length 7, otherwise fills with 0x55.
+    fn insecure_rng(dest: &mut [u8]) -> Result<(), Error> {
```

Comment:
I suggest `mock_` instead of `insecure_`.

Reply:
Not really a big deal since it was already named `insecure_` before.

Reply:
Renamed to `mock_rng` to better express our intent.